src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -12442,6 +12443,8 @@ void func_8084ECA4(Player* this, PlayState* play) {
         }
     }
 
+    //! @bug because func_8083C0E8 will change the animation and execution passes through here afterwards, this check
+    //! can pass for a single frame of the new animation the final time this function runs in a swing.
     if (this->skelAnime.curFrame <= 7.0f) {
```

Comment:
can add a sentence like "the end result is that the bottle will be active for a single frame long after the bottle was intended to be finished being active"

just to put it into perspective of what this really means